p2p/host/relay/autorelay.go

```diff
@@ -48,12 +50,13 @@ type AutoRelay struct {
 	cachedAddrsExpiry time.Time
 }
 
-func NewAutoRelay(ctx context.Context, bhost *basic.BasicHost, discover discovery.Discoverer, router routing.PeerRouting) *AutoRelay {
+func NewAutoRelay(ctx context.Context, bhost *basic.BasicHost, discover discovery.Discoverer, router routing.PeerRouting, static []peer.AddrInfo) *AutoRelay {
```

Comment:
This starts screaming "functional options".

Reply:
*shrug*

Reply:
it's not really user visible, so it doesn't offer much in terms of ergonomics.